pyccl/tests/test_parameters.py

```diff
@@ -99,7 +99,7 @@ def test_parameters_nu_list():
 
     assert np.allclose(cosmo['Omega_c'], 0.25)
     assert np.allclose(cosmo['Omega_b'], 0.05)
-    assert np.allclose(cosmo['Omega_m'], 0.30)
+    assert np.allclose(cosmo['Omega_m'] - cosmo['Omega_nu_mass'], 0.30)
```

Comment:
When the user defines a value for Omega_m in the cosmology object, are they defining an Omega_m that includes massive neutrinos or not? I would assume that with the new approach the answer would be yes.

Reply:
They don't ever do this. Omega_m value is computed.